packages/gatsby-theme-store/src/templates/product.tsx

```diff
@@ -109,13 +105,9 @@ const Page: FC<ProductPageProps> = (props) => {
 }
 
 export const query = graphql`
-  query ProductPageQuery(
-    $slug: String
-    $staticPath: Boolean!
-    $regionId: String
-  ) {
+  query ProductPageQuery($slug: String, $staticPath: Boolean!) {
     vtex {
-      product(slug: $slug, regionId: $regionId) @include(if: $staticPath) {
+      product(slug: $slug) @include(if: $staticPath) {
```

Comment:
This is interesting, how is the product page responding to region changes now?

Reply:
Yes so. This is the static data. regionalized data should be fetched via `useAsyncProduct` hook